src/huggingface_hub/_inference_types.py

```diff
@@ -0,0 +1,70 @@
+from typing import TYPE_CHECKING, List
+
+from .utils._typing import TypedDict
+
+
+if TYPE_CHECKING:
+    from PIL import Image
+
+
+class ClassificationOutput(TypedDict):
+    """Dictionary containing the output of a 'classification' task.
```

Comment:
Might be nice to be able to link back to the specific task :)

```suggestion
    """Dictionary containing the output of a [`~InferenceClient.audio_classification`] and  [`~InferenceClient.image_classification`] task.
```